.gitlab-ci.yml

```diff
@@ -435,11 +435,37 @@ check-polkadot-companion-build:
     - job:                         test-linux-stable-int
       artifacts:                   false
   script:
-    - ./.maintain/gitlab/check_polkadot_companion_build.sh
+    - ./.maintain/gitlab/check_companion_build.sh paritytech polkadot
   after_script:
     - cd polkadot && git rev-parse --abbrev-ref HEAD
   allow_failure:                   true
 
+check-cumulus-companion-build:
```

Comment:
Would something like this work? To avoid copy-and-pasting the same code

```yaml
.check-companion-build: &check-companion-build
  stage:                           build
  <<:                              *docker-env
  <<:                              *test-refs-no-trigger
  needs:
    - job:                         test-linux-stable-int
      artifacts:                   false
  script:
    - args=("$COMPANION_ORG" "$COMPANION_REPO")
    - if [ "${COMPANION_BUILD:-}" ]; then args+=("$COMPANION_BUILD"); fi
    - ./.maintain/gitlab/check_companion_build.sh "${args[@]}"
  after_script:
    - cd "$COMPANION_REPO" && git rev-parse --abbrev-ref HEAD

check-cumulus-companion-build:
  variables:
    COMPANION_ORG: paritytech
    COMPANION_REPO: cumulus
    COMPANION_BUILD: "cargo check"
  <<: *check-companion-build

check-polkadot-companion-build:
  variables:
    COMPANION_ORG: paritytech
    COMPANION_REPO: polkadot
  <<: *check-companion-build
```

This is very minor. Feel free to disregard if you don't think it's worth it.

Reply:
Good idea, cheers :) smart move. I've added this but didn't use arrays for the args - I don't believe it's guaranteed `script` runs in a bash shell, so I altered it to make it POSIX-compliant (arrays are not)